README.rst

```diff
@@ -223,11 +359,97 @@ Configure celery (you may use a different broker if you want):
         },
     }
 
-If you decide to use redis (as shown in these examples),
-install the requierd python packages::
+    INSTALLED_APPS.append('djcelery_email')
+    EMAIL_BACKEND = 'djcelery_email.backends.CeleryEmailBackend'
+
+If you decide to use Redis (as shown in these examples),
+install the following python packages.
+
+.. code-block:: shell
 
     pip install redis django-redis
 
+
+Quickstart Guide
+----------------
+
+Install OpenWISP Monitoring
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Install *openwisp-monitoring* from one of the methods mentioned in the
+`"Installation instructions" <#installation-instructions>`_ section.
+
+Install openwisp-config on the device
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the `installation instructions from openwisp-config documentation <https://github.com/openwisp/openwisp-config#install-precompiled-package>`_
+and configure the openwisp-config agent on the device.
+
+Set the ``management_interface`` in openwisp-config's configuration on the device.
+This interface will be used by OpenWISP to reach the device.
+
+E.g., if the *management interface* is ``tun0`` on the device, the openwisp-config
+configuration should look similar to the below example:
+
+.. code-block:: text
+
+    # In /etc/config/openwisp on the device
+    config controller 'http'
+        # other configuration directives
+        option management_interface 'tun0'
+
+The different modules of OpenWISP use the *management interface* for providing the following features:
+
+- `Pushing configuration changes <https://github.com/openwisp/openwisp-controller#how-to-configure-push-updates>`_
+  from OpenWISP to the device.
+- `Executing commands remotely on the device <https://github.com/openwisp/openwisp-controller#sending-commands-to-devices>`_.
+- `Running checks on the device <https://github.com/openwisp/openwisp-monitoring#available-checks>`_.
+- `Performing firmware upgrades on the device <https://github.com/openwisp/openwisp-firmware-upgrader#perform-a-firmware-upgrade-to-a-specific-device>`_
+
+It is **required** to set up the *management interface* for functioning
+of above-listed features.
+
+You can only get away from setting up a management interface
+if OpenWISP is deployed in a Layer 2 network, i.e. OpenWISP can reach the devices
+on their **Last IP**. In this scenario, set the `"OPENWISP_MONITORING_MANAGEMENT_IP_ONLY" <#openwisp-monitoring-management-ip-only>`_
+setting in OpenWISP to ``False``.
+
+Install monitoring packages on the device
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the `instructions from openwrt-openwisp-monitoring documentation <https://github.com/openwisp/openwrt-openwisp-monitoring/tree/master#install-pre-compiled-packages>`_
```

Comment:
please try to break the links to avoid having lines that are too long, eg:

```
`instructions ....
<link>`_
```

This helps with the maintenance of the document.